docs/changelogs/v2.5.0.md

```diff
@@ -0,0 +1,117 @@
+## Changelog
+
+### Features
+
+- Templates can now be deprecated in "template settings" to warn new users and prevent new workspaces from being created (#10745) (@Emyrk)
+  ![Deprecated template](https://gist.github.com/assets/22407953/5883ff54-11a6-4af0-afd3-ad77be1c4dc2)
+  > This is an [Enterprise feature](https://coder.com/docs/v2/latest/enterprise).
+- Add user/settings page for managing external auth (#10945) (@Emyrk)
+  ![External auth settings](https://gist.github.com/assets/22407953/99252719-7255-426e-ba88-55d08dd04586)
+- Allow auditors to read template insights (#10860) (@johnstcn)
+- Add support for custom permissions in Helm chart `rbac.yaml` file (#10590) (@lbi22)
+- Add `workspace_id`, `owner_name`` to agent manifest (#10199) (@szab100)
+- Allow identity provider to return single string for roles/groups claim (#10993) (@Emyrk)
+- Add endpoints to list all auth-ed external apps (#10944) (@Emyrk)
+- Support v2 Tailnet API in AGPL coordinator (#11010) (@spikecurtis)
+- Dormant workspaces now appear in the default workspaces list (#11053) (@sreya)
+- Include server agent API version in buildinfo (#11057) (@spikecurtis)
+- Restart stopped workspaces on `coder ssh` command (#11050) (@Emyrk)
+- Add group allowlist for oidc (#11070) (@Emyrk)
+- Display 'Deprecated' warning for agents using old API version (#11058) (@spikecurtis)
+- Add support for `coder_env` resource to set environment variables within modules (#11102) (@mafredri)
```

Comment:
```suggestion
- Add support for `coder_env` resource to set environment variables within a workspace (#11102) (@mafredri)
```
It can be used outside of modules too.